apps/expo-go/README.md

```diff
@@ -0,0 +1,40 @@
+# Expo Go
+
+For external contributors: If you want to contribute to the Expo SDK, use the Bare Expo app for developing and testing your changes (unless your changes are specific to the Expo Go app).
+
+## Building Expo Go
+
+1. Build React Native
+
+You can build the React Native Android dep using `./gradlew :packages:react-native:ReactAndroid:buildCMakeDebug` in `react-native-lab/react-native` directory. This is optional because React Native will be built anyway when you build Expo Go, but can help to narrow down a potential issue surface area. 
```

Comment:
wondering if this should be put in the Troubleshooting section? also we can run `./gradlew :packages:react-native:ReactAndroid:buildCMakeDebug` in `apps/expo-go/android` as well.